frameworks/helloworld/tests/test_overlay.py

```diff
@@ -111,3 +121,36 @@ def test_overlay_network():
     assert "dns" in host_endpoints_result.keys()
     assert len(host_endpoints_result["dns"]) == 1
     assert host_endpoints_result["dns"][0] == hosts.autoip_host(PACKAGE_NAME, "hello-host-vip-0-server", 4044)
+
+
+@pytest.mark.sanity
+@pytest.mark.overlay
+def test_port_names():
+    def check_task_ports(task_name, expected_port_count, expected_port_names):
+        endpoint = "/v1/tasks/info/{}".format(task_name)
+        try:
+            r = api.get(PACKAGE_NAME, endpoint).json()
+        except DCOSHTTPException:
+            return False, "Failed to get API endpoint {}".format(endpoint)
+        networks.check_port_names(r, expected_port_count, expected_port_names)
+
+    for task in TASKS_WITH_PORTS:
+        if task == "hello-overlay-0-server":
+            check_task_ports(task, 2, ["dummy", "dynport"])
+        else:
+            check_task_ports(task, 1, ["test"])
+
+@pytest.mark.sanity
+@pytest.mark.overlay
+def test_srv_records():
+    fmk_srvs = networks.get_framework_srv_records(PACKAGE_NAME)
+    for task in TASKS_WITH_PORTS:
+        task_records = networks.get_task_record(task, fmk_srvs)
+        if task == "hello-overlay-0-server":
+            assert len([r for r in task_records if "dummy" in r["name"]]) == 1, "Missing SRV record for dummy and "\
```

Comment:
Whoops, misread the formatting. It could be, though, for pseudo-PEP8 compliance :)